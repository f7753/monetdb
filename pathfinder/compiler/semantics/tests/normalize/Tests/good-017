#!/bin/bash

pf -s2 <<'EOT'
(: Note that 'encoding' is a path expression :)
xquery version "1.0";
encoding
EOT
echo $?
