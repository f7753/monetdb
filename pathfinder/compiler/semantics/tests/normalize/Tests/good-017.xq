(: Note that 'encoding' is a path expression :)
xquery version "1.0";
encoding
