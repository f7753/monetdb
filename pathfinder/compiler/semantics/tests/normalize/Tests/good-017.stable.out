stdout of test 'good-017` in directory 'compiler/semantics/tests/normalize` itself:


# 13:31:27 >  
# 13:31:27 >  Mtimeout -timeout 60 ./good-017 good-017 
# 13:31:27 >  

0

# 13:31:27 >  
# 13:31:27 >  Done.
# 13:31:27 >  

